peg-tests/Cargo.toml

```diff
@@ -7,5 +7,8 @@ repository = "https://github.com/kevinmehall/rust-peg"
 description = "Tests for the peg crate"
 build = "build.rs"
 
+[dependencies]
+peg-syntax-ext = { version = "0.5", path = "../peg-syntax-ext" }
```

Comment:
The issue remains that I couldn't include another grammar at build-time due to name conflicts in all the boilerplate that was being included.